doc/src/tutorials/physics/quantum/index.rst

```diff
@@ -0,0 +1,18 @@
+.. _quantum_mechanics_tutorial:
+
+=================================
+Quantum Mechanics Tutorials
+=================================
+
+These tutorials by Dr. Al Rey Villagracia demonstrate how to use the sympy.physics.quantum module and related
```

Comment:
Please remove your name here. We don't attribute others in parts of SymPy documents like this. The  implicit attribution is to all SymPy authors.